terraform/environment/terraform.tfvars.json

```diff
@@ -16,6 +16,7 @@
       "is_production": "false",
       "sirius_api_gateway_endpoint": "https://api.dev.sirius.opg.digital/v1/lpa-online-tool/lpas/",
       "sirius_api_gateway_arn": "arn:aws:execute-api:eu-west-1:288342028542:*/*/GET/lpa-online-tool/*",
+      "auth_token_ttl_secs": 330,
```

Comment:
thinking - is 330 seconds too long?

Reply:
From the perspective of the tests, they have to wait 35 seconds before the session timeout popup appears. It seems OK to me. A bigger concern is whether this is too short when you're developing, as you get timed out (locally) every 30 seconds.